app/src/main/java/org/wikipedia/activity/BaseActivity.kt

```diff
@@ -239,7 +239,10 @@ abstract class BaseActivity : AppCompatActivity() {
                     .setCancelable(false)
                     .setTitle(R.string.logged_out_in_background_title)
                     .setMessage(R.string.logged_out_in_background_dialog)
-                    .setPositiveButton(R.string.logged_out_in_background_login) { _: DialogInterface?, _: Int -> startActivity(LoginActivity.newIntent(this@BaseActivity, LoginFunnel.SOURCE_LOGOUT_BACKGROUND)) }
+                    .setPositiveButton(R.string.logged_out_in_background_login) { dialog: DialogInterface, _: Int ->
+                        startActivity(LoginActivity.newIntent(this@BaseActivity, LoginFunnel.SOURCE_LOGOUT_BACKGROUND))
+                        dialog.dismiss()
```

Comment:
This should not be necessary.  An AlertDialog will automatically dismiss itself when either the Positive or Negative button is clicked.